### PR TITLE
Properties page

### PR DESCRIPTION
**Pull request by galvana**
Closes [PROD-1661](https://ethyca.atlassian.net/browse/PROD-1661), [PROD-1662](https://ethyca.atlassian.net/browse/PROD-1662), [PROD-1663](https://ethyca.atlassian.net/browse/PROD-1663)

### Description Of Changes

Updating navbar and adding first iteration of the properties page with the ability to create a property with a name, type (Website & Other), and an auto-generated key.

### Code Changes

* [ ] Removed "Classify systems" page
* [ ] Updated icons for Copy and Delete actions
* [ ] Added new `CustomClipboardCopyProps` form component
* [ ] New properties page and table
* [ ] New `properties.slice.ts`
* [ ] Moved `EmptyTableNotice` component to render inside the table
* [ ] New property table, model, and schema
* [ ] Cypress test for properties page

### Steps to Confirm

* [ ] Run `nox -s "build(slim)" "dev(slim)" -- dev` against the fidesplus branch found here https://github.com/ethyca/fidesplus/pull/1326
* [ ] Run `turbo run dev` in the `clients` directory of `fides`
* [ ] Navigate to the Properties page and verify you can create and list properties

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

[PROD-1661]: https://ethyca.atlassian.net/browse/PROD-1661?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PROD-1662]: https://ethyca.atlassian.net/browse/PROD-1662?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PROD-1663]: https://ethyca.atlassian.net/browse/PROD-1663?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts [hunk 1]
```diff
@@ -155,19 +155,19 @@ describe("Config wizard with plus settings", () => {
         const { body } = interception.request;
         expect(body.length).to.eql(numSystems);
       });
-      cy.getByTestId("systems-classify-table");
-      cy.url().should("contain", "classify-systems");
+      // cy.getByTestId("systems-classify-table");
+      // cy.url().should("contain", "classify-systems");
```
Comment (galvana) [inline clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts:159]: These are tests for the "data flow scanner" and they verify functionality by navigating to the "Classify systems" page. However, @mfbrown asked that I remove the "Classify systems" page from the navbar as part of this ticket. For now I just removed the assertions that occur after navigating to the system-classify-table.

File: clients/admin-ui/cypress/e2e/systems-classify.cy.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { stubPlus, stubTaxonomyEntities } from "cypress/support/stubs";
 
-describe("Classify systems page", () => {
+describe.skip("Classify systems page", () => {
```
Comment (galvana) [inline clients/admin-ui/cypress/e2e/systems-classify.cy.ts:3]: Skipping all of the tests for the Classify systems page

File: clients/admin-ui/src/features/common/form/inputs.tsx [hunk 3]
```diff
@@ -1190,3 +1192,65 @@ export const CustomCheckbox = ({
...
   );
 };
+
+interface CustomClipboardCopyProps {
+  label?: string;
+  tooltip?: string;
+  variant?: Variant;
+}
+
+export const CustomClipboardCopy = ({
```
Comment (galvana) [inline clients/admin-ui/src/features/common/form/inputs.tsx:1202]: New form field component for the clipboard copy functionality
image

File: clients/admin-ui/src/features/common/nav/v2/BackButton.tsx [hunk 4]
```diff
@@ -15,9 +15,7 @@ const BackButton = ({
   backPath,
   ...props
 }: { backPath: string } & FlexProps) => (
-  // In general Layout provides a default top padding. But on pages with
-  // Back buttons, we want the Back button to be higher and for the page to be less top-padded
-  <Flex alignItems="center" mt={-4} mb={3} {...props}>
+  <Flex alignItems="center" mb={6} {...props}>
```
Comment (galvana) [inline clients/admin-ui/src/features/common/nav/v2/BackButton.tsx:18]: Opinionated change here. I know the intention was to make this less "top-padded" but I didn't think it looked good for the back button to be so close to the top of the page, this is what it looks like now.
image

File: clients/admin-ui/src/features/common/table/v2/FidesTable.tsx [hunk 5]
```diff
@@ -119,6 +121,7 @@ const TableBody = <T,>({
   onRowClick,
   renderRowTooltipLabel,
   displayAllColumns,
+  emptyTableNotice,
```
Comment (galvana) [inline clients/admin-ui/src/features/common/table/v2/FidesTable.tsx:124]: Updated `FidesTable` to accept an `emptyTableNotice` component to display inside the table body. I updated the Properties and Add Vendor tables to use this new design.
image

image

File: src/fides/api/models/property.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+from fides.api.db.base_class import Base
+from fides.api.db.util import EnumColumn
+from fides.api.schemas.property import PropertyType
+from fides.config import get_config
+
+CONFIG = get_config()
+
+
+class Property(Base):
```
Comment (adamsachs) [inline src/fides/api/models/property.py:14]: given how generic and overloaded a term Property is (i know, not your choice!), and also how generic the fields/columns are for this table, i'd definitely include a docstring here to give more context on what we mean by Property in this class/table!

i'll let you come up with the wording for that 😅 but happy to give a review or brainstorm with ya if you're stuck

File: src/fides/api/models/property.py [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+    @declared_attr
+    def __tablename__(self) -> str:
+        return "plus_property"
+
```
Comment (adamsachs) [inline src/fides/api/models/property.py:18]: nit: given my [comment](https://github.com/ethyca/fidesplus/pull/1326#discussion_r1503232707) on the plus PR re: duplication error handling, you _could_ override the `create` method of the base class and handle "name" and "key" duplications in a way here that makes it clearer to report on them in that endpoint function.

not a must-have, just thinking about how to make that error handling as simple as possible while providing the user with sensible informtation!

**Review by adamsachs — APPROVED** [review]
this looks good to me, only a couple of minor points!

**Review by Kelsey-Ethyca — CHANGES_REQUESTED** [review]
Left some comments following ticket AC functionality / designs. The code structure is good!

File: src/fides/api/models/property.py (hunk 7 above)
Comment (galvana) [reply src/fides/api/models/property.py:22]: I think for now I'll just separate `KeyOrNameAlreadyExists` exceptions from other exceptions and keep this fancier check for a future iteration

**Review by galvana — COMMENTED** [review]
Thank you @Kelsey-Ethyca and @adamsachs for the first pass! I've addressed all of your comments, please let me know what you think

**Review by adamsachs — APPROVED** [review]
looks good to me on the BE, thanks for making the suggested tweaks 👍 

just a reminder to update the changelog (either here or on the plus PR)!

**Review by Kelsey-Ethyca — APPROVED** [review]
Lovely ❤️

**Pull request merged**